src/recognition/recognition.service.ts

```diff
@@ -64,16 +64,40 @@ export class RecognitionService {
       companyValues = [],
       giphyUrl,
       media = [],
+      departments = [],
     }: CreateRecognitionDto,
   ) {
     const invalidValues = companyValues.filter(
       (value) => !Object.values(CompanyValues).includes(value),
     );
 
     if (receivers.length === 0) {
-      throw new BadRequestException(
-        'At least one receiver is required for recognition',
-      );
+      if (!departments || departments.length === 0) {
```

Comment:
make this more readable 
```suggestion
      if (!receivers?.length  || !departments.length ) 
      // return error
```